numpy/core/tests/test_regression.py

```diff
@@ -2368,6 +2368,11 @@ def test_void_item_memview(self):
         del va
         assert_equal(x, b'\x00\x00\x00\x00')
 
+    def test_void_getitem(self):
+        # Test fix for gh-11668.
+        for x in [asbytes(a) for a in 'abcd']:
+            assert_(np.array([x], 'V1').astype('O') == x)
```

Comment:
I think this test needs multiple items in the array to fail

Reply:
No, I checked that.
```
In [1]: from numpy.compat import asbytes

In [2]: for x in [asbytes(a) for a in 'abcd']:
   ...:     assert np.array([x], 'V1').astype('O') == x
   ...:     
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
<ipython-input-2-b41e23938857> in <module>()
      1 for x in [asbytes(a) for a in 'abcd']:
----> 2     assert np.array([x], 'V1').astype('O') == x
      3 

AssertionError: 

```

Reply:
Or maybe this is more convincing
```
In [3]: for x in [asbytes(a) for a in 'abcd']:
   ...:     print(np.array([x], 'V1').astype('O'), x)
   ...:     
   ...:     
[b'\x00'] b'a'
[b'\x00'] b'b'
[b'\x00'] b'c'
[b'\x00'] b'd'
```